packages/preferences/src/browser/preference-frontend-module.ts

```diff
@@ -30,6 +30,7 @@ import './preferences-monaco-contribution';
 export function bindPreferences(bind: interfaces.Bind, unbind: interfaces.Unbind): void {
     unbind(PreferenceProvider);
 
+    bindDefaultPreferenceProvider(bind);
```

Comment:
why do we need an external function
seems like the two following are not using a function so maybe could just write there:

```
bind(PreferenceProvider).to(PreferenceSchemaProvider).inSingletonScope().whenTargetNamed(PreferenceScope.Default);
```

Reply:
I added this function to avoid further merge conflict with https://github.com/theia-ide/theia/pull/3247.